### PR TITLE
Manually set textblock color for audio articles

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

Creates a palette colour for the textblock text and implements it.

## Why?

Currently textblock text colour is inherited rather than set.

## Screenshots

| Before      | After      |
| ----------- | ---------- |
| ![before][] | ![after][] |

[before]: https://github.com/user-attachments/assets/366a1fe2-bb3b-4c2f-a6b2-0ec653a77800
[after]: https://github.com/user-attachments/assets/8b9dba7e-e190-4260-ace9-cd3f5bd4e5ab

### CODE REVIEW

File: dotcom-rendering/src/palette.ts [hunk 1]
```diff
@@ -49,6 +49,24 @@ const pillarPalette = (
 	}
 };
 
+const textblockTextLight: PaletteFunction = (format: ArticleFormat) => {
+	switch (format.design) {
+		case ArticleDesign.Audio:
+			return sourcePalette.neutral[97];
+		default:
+			return sourcePalette.neutral[7];
```
Comment (abeddow91) [inline dotcom-rendering/src/palette.ts:57]: As discussed, a safer default here might be
```suggestion
			return 'inherit';
```

**Review by domlander — CHANGES_REQUESTED** [review]
There looks to be an issue in at least one of the chromatic diffs (paragraph 7 and 8): https://www.chromatic.com/test?appId=63e251470cfbe61776b0ef19&id=66d999e6efdcb59f01e473e0

**Comment by domlander** [comment]
> There looks to be an issue in at least one of the chromatic diffs (paragraph 7 and 8): https://www.chromatic.com/test?appId=63e251470cfbe61776b0ef19&id=66d999e6efdcb59f01e473e0

This might require an update to the story rather than the implementation. We get [all designs for the story](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/KeyTakeaways.stories.tsx#L85), which includes audio, for the story. Now that audio has a dark background, maybe separate out the audio design from this story and create a new story? This is just a suggestion and if you have any other ideas then please go for it!

**Comment by joecowton1** [comment]
> > There looks to be an issue in at least one of the chromatic diffs (paragraph 7 and 8): [chromatic.com/test?appId=63e251470cfbe61776b0ef19&id=66d999e6efdcb59f01e473e0](https://www.chromatic.com/test?appId=63e251470cfbe61776b0ef19&id=66d999e6efdcb59f01e473e0)
> 
> This might require an update to the story rather than the implementation. We get [all designs for the story](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/KeyTakeaways.stories.tsx?rgh-link-date=2024-09-05T12%3A06%3A52Z#L85), which includes audio, for the story. Now that audio has a dark background, maybe separate out the audio design from this story and create a new story? This is just a suggestion and if you have any other ideas then please go for it!

Thanks @domlander will have a look at options!

**Comment by joecowton1** [comment]
@domlander I've removed the audio articles from key takeaways and mini profiles as I don't believe there's likely to be such examples in real life - let me know if you disagree.

**Review by domlander — APPROVED** [review]

**Pull request merged**